### ISSUE TITLE
subdirectory install sets wrong base href in editor

### ISSUE THREAD

**Issue by toderash** [andrew-worsfold/tailor#79]
/ is reachable but wp install is in /subdir/
When opening the Tailor editor, the base href="../../" and all the enqueued js & css files fail to load (404). This should be set to "../" or better still, "//qualified.domain.com/subdir/".
In this state, Tailor always attempts to edit the default home page.

**Comment by andrew-worsfold** [comment]
@toderash -

Which scripts are failing to load for you?

Tailor loads stylesheets and JavaScript using in the way recommended by WordPress (e.g., wp_enqueue_script() referring the plugin url obtained via plugin_dir_url()).  My test WordPress instances are also in subdirectories and do not suffer from this problem, which makes me wonder what change actually needs to be made.

Do you happen to know what specific code changes are required here?

Many thanks,
Andrew.

**Comment by toderash** [comment]
@andrew-worsfold thanks for the reply.
For now I simply hardcoded the value I need, and noted that the url was built the same way I would have done it - I tried using php superglobals and got the same result, so I think it has to do with minifying my html, so I'll have to assume this one is not a bug.

btw, most of the concatenation & minification attempts I've tried for js have also broken the editor, so I'm still looking for the right combination there.
